usaspending_api/etl/es_settings.json

```diff
@@ -6,10 +6,10 @@
 		}
 	},
 	"settings": {
-		"index.max_result_window" : 500000,
-		"index": {
-			"number_of_replicas": 1,
-			"refresh_interval": "1s"
-		}
+		"index.max_result_window" : null
```

Comment:
Does this always get overwritten? Seems dangerous...

Reply:
It will be overwritten by the script that parses the json.

I only know of three options (if you have others please share):

1. Duplicate hardcoded values in different files (what is currently done)
2. Move the value to the centralized Python settings file and populate the json (what the PR code does)
3. Store the value here and read the file for settings.py

Since the es_settings.json file is already being updated by the script in the current workflow, this isn't changing much.

Reply:
4. Don't keep `es_settings.json`, move the JSON to be assembled by the script. Effectively storing the same info as a python dictionary populated with settings vars on creation

Reply:
Can you point me to the script that modifies this? I don't like the sound of a mutable settings file, but I'd want to look at what's doing this before I can come up with useful suggestions.

Reply:
Ok, it looks like just es_etl_helpers. I'm going to read through it more, but I'm confused where the duplication is coming from. Even if there is a reason it needs to be modified in more than one place, could we structure the json to show the seperation? If I see something set to `null` in a settings file, my first instinct is that someone wanted that value null, not that it's being changed behind the scenes.

Reply:
Would it be better if `es_settings.json` was renamed to `es_settings_template.json`?

Reply:
Throw in a comment explaining the situation and I'd call that a good compromise to get this finished.

Reply:
Where?

Reply:
Right, this is a JSON file. I guess the rename will have to do, since any comment outside the file is less likely to be seen than the code that shows what happens directly. After that I'd be ready to approve.

Reply:
The Python library cannot parse a JSON file with comments. Comments are not part of the JSON spec. This [SO answer](https://stackoverflow.com/a/41903882) includes a comment on why they are not in the spec

Reply:
That's what I was trying to say. I had my struggle with JSON and comments long ago; just renaming the file is the best hint I think we can give to future devs.